tests/filecheck/dialects/arith/arith_ops.mlir

```diff
@@ -164,4 +164,10 @@
 
   // CHECK-NEXT: %selecti = arith.select %lhsi1, %lhsi32, %rhsi32 : i32
   // CHECK-NEXT: %selectf = arith.select %lhsi1, %lhsf32, %rhsf32 : f32
+
+  %sum, %carry = arith.addui_extended %lhsi32, %rhsi32 : i32, i1
+  %sum_index, %carry_index = arith.addui_extended %lhsi64, %rhsi64 : i64, i1
```

Comment:
```suggestion
  %sum, %carry = arith.addui_extended %lhsi32, %rhsi32 : i32, i1
  %sum_index, %carry_index = arith.addui_extended %lhsi64, %rhsi64 : i64, i1
```
Shouldn't these be generic?

Reply:
Half of this file has custom syntax.
Unsure then what we should have there?

Reply:
I personally don't think we need checks for generic syntax but I don't feel strongly about any of this, just thought I'd bring it up to make sure you're aware